Ubuntu_Dockerfile

```diff
@@ -61,11 +61,13 @@ WORKDIR /root
 RUN set -x && \
 # Install prerequirements
     DEBIAN_FRONTEND=noninteractive apt-get -yq install curl \
-    dkms make autoconf autotools-dev chrpath automake hostname debhelper gcc quilt libc6-dev build-essential pkg-config && \
+    dkms make autoconf autotools-dev chrpath automake hostname debhelper gcc-12 quilt libc6-dev build-essential pkg-config && \
 # Cleanup
     apt-get clean autoclean && \
     rm -rf /var/lib/apt/lists/*
 
+RUN ln -fs gcc-12 /usr/bin/gcc  # 'build-essential' installs `gcc`, however we need `gcc-12`, so we overwrite it with this link
```

Comment:
What is the original issue?
Does it impact "sources" flow?

Reply:
It resolved a compilation issue. Tzafrir would be able to provide more information if required...